application/language/spanish/station_lang.php

```diff
@@ -46,72 +46,72 @@
-$lang['station_location_oqrs_enabled'] = "OQRS Enabled";
-$lang['station_location_oqrs_email_alert'] = "OQRS Email alert";
-$lang['station_location_oqrs_email_hint'] = "Make sure email is set up under admin and global options.";
-$lang['station_location_oqrs_text'] = "OQRS Text";
-$lang['station_location_oqrs_text_hint'] = "Some info you want to add regarding QSL'ing.";
-$lang['station_location_clublog_realtime_upload']='ClubLog Realtime Upload';
+$lang['station_location'] = 'Localización de Estación';
+$lang['station_location_plural'] = "Localizaciones de Estaciones";
+$lang['station_location_header_ln1'] = 'Las localizaciones de Estación le permiten definir localizaciones de operación, como su QTH, el QTH de un amigo o una estación portátil.';
+$lang['station_location_header_ln2'] = 'De forma similar a los libros de guardia, un perfil de estación mantiene asociado un conjunto de QSOs.';
+$lang['station_location_header_ln3'] = 'Solo una estación puede estar activa en cualquier momento. En la tabla de abajo esta se muestra con la etiqueta de -Estación Activa-.';
+$lang['station_location_create_header'] = 'Crear Localización de Estación';
+$lang['station_location_create'] = 'Crear una Localización de Estación';
+$lang['station_location_edit'] = 'Editar Localización de Estación: ';
+$lang['station_location_updated_suff'] = ' Actualizada.';
+$lang['station_location_warning'] = 'Atención: Debe configurar una Localización de Estación como activa. vaya a Indicativo->Localización de Estación para seleccionar una.';
+$lang['station_location_reassign_at'] = 'Por favor, reasignelas en ';
+$lang['station_location_warning_reassign'] = 'Debido a cambios recientes en Cloudlog, debe reasignar sus QSO a sus perfiles de estación.';
+$lang['station_location_name'] = 'Nombre de Perfil';
+$lang['station_location_name_hint'] = 'Nombre corto para la Localización de Estación. Ejemplo: Casa (IO87IP)';
+$lang['station_location_callsign'] = 'Indicativo de la Estación';
+$lang['station_location_callsign_hint'] = 'Indicativo de llamada de la Estación. Ejemplo: 2M0SQL/P';
+$lang['station_location_power'] = 'Potencia de la Estación (W)';
+$lang['station_location_power_hint'] = 'Potencia de la estación por defecto en Vatios. Puede ser sobreescrito por CAT.';
+$lang['station_location_emptylog'] = 'Libro Vacío';
+$lang['station_location_confirm_active'] = '¿Está seguro que desea poner como activa la siguiente estación: ';
+$lang['station_location_set_active'] = 'Poner como Activa';
+$lang['station_location_active'] = 'Estación Activa';
+$lang['station_location_claim_ownership'] = 'Reclamar Propiedad';
+$lang['station_location_confirm_del_qso'] = '¿Está seguro que desea eliminar todos los QSOs en este perfil de estación?';
+$lang['station_location_confirm_del_stationlocation'] = '¿Está seguro que desea eliminar el perfil de estación  ';
+$lang['station_location_confirm_del_stationlocation_qso'] = '? Esto eliminará todos los QSOs dentro este perfil de estación.';
```

Comment:
Why is the closing question mark in the second line rather than the first one?

Reply:
As in similar situations like this, Cloudlog uses these two lines to create a full string by adding the Station Profile name between the texts, i.e.: "_¿Está seguro que desea eliminar el perfil de estación_  ABCD Test_? Esto eliminará todos los QSOs dentro este perfil de estación._"

Reply:
Okay. Then this seems fine.